server/controllers/post-form-controller.js

```diff
@@ -1,9 +1,9 @@
 const main = (req, res) => {
 	const { name, email, country, gdpr } = req.body;
-	console.log('name:', name);
-	console.log('email:', email);
-	console.log('country:', country);
-	console.log('gdpr:', gdpr);
+	// console.log('name:', name);
+	// console.log('email:', email);
+	// console.log('country:', country);
+	// console.log('gdpr:', gdpr);
```

Comment:
do we need this commented code?

Reply:
Not really, it was only there to show what values were being used. I'll remove it now.